frontend/src/data/embroideries.ts

```diff
@@ -20,4 +20,6 @@ export const AllEmbroideries: Embroidery[] = [
     firstDorsalIntermetacarpalArteryGuildChrisWrightman,
     firstDorsalIntermetacarpalArteryMedAskaGhulam
 
-]
\ No newline at end of file
+]
+
+// build the list by importing the json file
```

Comment:
I think everything is good here apart from this - could you make sure all the embroideries get added to this list? Thank you!